<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Example2</class>
 <widget class="QWidget" name="Example2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Example 2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labelExampleTitle">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Get metadata from a file as JSON format</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelImgPathTxt">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Img path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditImgPath">
       <property name="text">
        <string>test.jpg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxExtractBinary">
       <property name="text">
        <string>Extrtact binary data</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBoxPrintTagID">
       <property name="text">
        <string>Print TagID (hex)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBoxPrintDescValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Print Tag desc + unconverted value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QPushButton" name="pushButtonRead">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEditStdOut">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Exiftool standart output (empty)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEditStdErr">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Exiftool error output (empty)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutStatus">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="labelInfoTxt">
       <property name="text">
        <string>Info:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProcessStatusTxt">
       <property name="text">
        <string>Process status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProcessStatus">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Not running</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditImgPath</tabstop>
  <tabstop>checkBoxExtractBinary</tabstop>
  <tabstop>checkBoxPrintTagID</tabstop>
  <tabstop>checkBoxPrintDescValue</tabstop>
  <tabstop>pushButtonRead</tabstop>
  <tabstop>textEditStdOut</tabstop>
  <tabstop>textEditStdErr</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditImgPath</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonRead</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
